packages/shared/src/lib/auth.ts

```diff
@@ -97,18 +133,16 @@ export interface AuthSession extends LogoutSessionData {
   issued_at: Date;
 }
 
-const authUrl = 'http://127.0.0.1:4433';
-
-interface LoginInitializationData extends InitializationData {
-  created_at: '2022-07-21T05:18:27.975693Z';
-  updated_at: '2022-07-21T05:18:27.975693Z';
-  refresh: false;
-  requested_aal: 'aal1';
+export interface SuccessfulRegistrationData {
+  session: AuthSession;
+  identity: Identity;
 }
 
+const authUrl = 'http://127.0.0.1:4433';
```

Comment:
Probably best to put this in a env actually.

Reply:
Totally agree. Let's just make this value a fallback if the env is empty. Thank you!